src/VisualStudio/Core/Def/InheritanceMargin/InheritanceMarginTaggerProvider.cs

```diff
@@ -54,6 +54,12 @@ internal sealed class InheritanceMarginTaggerProvider : AsynchronousViewportTagg
 
     protected override TaggerDelay EventChangeDelay => TaggerDelay.OnIdle;
 
+    /// <summary>
+    /// We support frozen partial semantics, so we can quickly get inheritance margin items without building SG docs.
+    /// We will still run a tagging pass after the frozen-pass where we run again on non-frozen docs.
+    /// </summary>
+    protected override bool SupportsFrozenPartialSemantics => true;
```

Comment:
this should also help with the current state of hte world where inheritance margin often doesn't show all downstream inherited items for things because the downstream projects it is looking at are never hydrated in the frozen partial solution.